docs/content/preview/explore/change-data-capture/using-logical-replication/overview.md

```diff
@@ -9,4 +9,195 @@ menu:
+
+In logical replication, the fundamental unit of data transmission is a transaction. A logical slot emits each change just once in normal operation. The current position of each slot is persisted only at checkpoint, so if a replication process is interrupted and restarts, even if the checkpoint or the starting LSN falls in the middle of a transaction, **the entire transaction is retransmitted**. This behavior guarantees that clients receive complete transactions without missing any intermediate changes, maintaining data integrity across the replication stream​. Logical decoding clients are responsible for avoiding ill effects from handling the same message more than once. Clients may wish to record the last LSN they saw when decoding and skip over any repeated data or (when using the replication protocol) request that decoding start from that LSN rather than letting the server determine the start point.
+
+### PUBLICATION
+
+A publication is a set of changes generated from a table or a group of tables, and might also be described as a change set or replication set. Each publication exists in only one database.
+
+Publications are different from schemas and do not affect how the table is accessed. Each table can be added to multiple publications if needed. Publications may currently only contain tables. Objects must be added explicitly, except when a publication is created for ALL TABLES.
+
+### OUTPUT PLUGIN
+
+Output plugins are the components used to decode WAL changes and transform them into a specific format that can be consumed by replication clients. These plugins are notified about the change events that need to be processed and sent via various callbacks. These callbacks are only invoked when the transaction actually commits.
+
+YugabyteDB supports the following four output plugins:
+* `yboutput`
+* `pgoutput`
+* `test_decoding`
+* `wal2json`
+
+All these plugins are pre-packaged with YugabyteDB and do not require any external installation.
+
+### LSN
+
+LSN in YugabyteDB is an unsigned 64-bit integer that uniquely identifies a change record or a transaction boundary record that is consumed from a given replication slot.
+
+In YugabyteDB, LSN values from different slots are considered unrelated and should not be compared. In YugabyteDB, LSN no longer represents the byte offset of a WAL record. 
+
+LSN values for a single replication slot satisfy the following properties: 
+
+- **Uniqueness** 
+
+  - LSN values for the change and transaction boundary records for a given replication slot are unique. In particular, changes from different tablets of the same table will have unique LSN values.
+
+* **Ordering** 
+
+  - LSN values can be compared ( <, >, = ).
+
+  - The LSN of the BEGIN record of a transaction will be strictly lower than the LSN of the changes in that transaction that in turn will be strictly lower than the LSN of the COMMIT record of the same transaction.
```

Comment:
The LSN of the change records in a transaction will be strictly lower than the LSN of the COMMIT record of the same transaction.